src/server/redis_connection.h

```diff
@@ -71,6 +71,12 @@ class Connection : public EvbufCallbackBase<Connection> {
   std::string Double(double d) const {
     return protocol_version_ == RESP::v3 ? "," + util::Float2String(d) + CRLF : BulkString(util::Float2String(d));
   }
+  // type should be "txt" or "mkd"
+  // if RESP is V2, treat verbatim string as blob string
+  std::string VerbatimString(const std::string &type, const std::string &data) const {
+    return protocol_version_ == RESP::v3 ? "=" + std::to_string(3 + data.size()) + CRLF + type + ":" + data + CRLF
+                                         : BulkString(data);
+  }
```

Comment:
```suggestion
  std::string VerbatimString(const std::string &type, const std::string &data) const {
    return protocol_version_ == RESP::v3 ? "=" + std::to_string(type.size() + data.size()) + CRLF + type + ":" + data + CRLF
                                         : BulkString(data);
  }
```
Even the length of `type` *should* be 3, but there is no check so if we pass a wrong type it will be a big issue.

Reply:
I think this function should check `type` is 'txt' or 'mkd', but i don't know how to handle the wrong type. Throw an error??

Reply:
Currently I think you don't need to check. Or you can use the `CHECK` macro from glog.

Reply:
Ok, thanks